types/email-templates/index.d.ts

```diff
@@ -103,11 +112,26 @@ declare namespace EmailTemplate {
          * @param view The Html pug to render
          * @param locals The template Variables
          */
-        function render(view: string, locals: any): Promise<string>;
+        function render(view: string, locals?: any): Promise<string>;
+
+        /**
+         * Render all available template files for a given email
+         * template (e.g. `html.pug`, `text.pug`, and `subject.pug`)
+         *
+         * @param view Name of the template
+         * @param locals The template variables
+         */
+        function renderAll(view: string, locals?: any): Promise<EmailMessage>;
 
         /**
          * Send the Email
          */
         function send(options: EmailOptions): any;
+
```

Comment:
Interface EmailMessage should be declared with the rest of the interfaces.

Reply:
Not sure how to do this … if I simply move it to the upper level, it seems I can no longer access it from outside?

Reply:
You should be able to access from the outside, just like the other interfaces.